From d27c624fca748b1b257d04a72d46cdcbf6775b01 Mon Sep 17 00:00:00 2001
From: Roman Li <Roman.Li@amd.com>
Date: Thu, 15 Dec 2016 11:51:09 -0500
Subject: [PATCH 1843/3082] drm/amd/display: fix hotplug regression after code
 refactor

The condition logic of REG_WAIT in dce110_stream_encoder_dp_blank()
got inverted after refactoring.

Change-Id: Idf4cece8f49a3bad51b352c3155f8ac8cb0b4907
Signed-off-by: Roman Li <Roman.Li@amd.com>
Reviewed-by: Jordan Lazare <Jordan.Lazare@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c b/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
index 0590e0a..b74a29b 100644
--- a/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
+++ b/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
@@ -635,7 +635,7 @@ static void dce110_stream_encoder_dp_blank(
 	*/
 
 	REG_WAIT(DP_VID_STREAM_CNTL, DP_VID_STREAM_STATUS,
-			1,
+			0,
 			10, max_retries);
 
 	ASSERT(retries <= max_retries);
-- 
2.7.4

